### PR TITLE
Update to QGL locations, no probe offset

### PR DESCRIPTION
**Pull request by morph5555**
Update to QGL locations, no probe offset

### CODE REVIEW

**Review by nemgrea — APPROVED** [review]

**Comment by kyleisah** [comment]
Shouldn't we also have Trident's positions for z_tilt in here as well? Also, might be a good idea to clarify the values being edited. Currently it looks like you're just uncommenting something in the [probe] section when they should be in a different section related to leveling/trimming.

**Comment by morph5555** [comment]
Sure I'll take a look at the Trident's config. and post a reply here so I
can get comments on it. I don't own a Trident, but should be easy enough to
sort out since the mechanics are known.

**Comment by raymondh2** [comment]
What's the issue with leaving the probe points as is?

**Comment by nemgrea** [comment]
> What's the issue with leaving the probe points as is?

the standard QGL has an offset built in because our probe was behind the nozzle when using an inductive setup. tap is using the actual nozzle so that offset should be 0

**Comment by morph5555** [comment]
The first issue is consistency of a square object being probed squarely to
get an accurate horizontal plane.

TLDR- Garbage in Garbage out:
The second issue is for the kinematic mounts, it seems with limited data on
my end that moving the probing points back and out farther and keeping them
consistent is providing more accurate reads when it comes to build plate
deflection even if it's ever so slight. I measured my MRW kinematic bed
holding force @ 996 grams at 25 mm from a true 350mm build plate's edge. I
only used a laser across the build surface to look for deflection while
probing ( rudimentarily ) visually surveying for movement. That being said
I am certain there is movement long before I could see it. Not the most
accurate way to test for deflection but the best way that met both my time
table and parts on hand.

Spent a lot of time building a very accurate probe, should probably use it
accordingly.

**Comment by skyaee** [comment]
Shouldn't we also have Trident's positions for z_tilt in here as well?

**Comment by morph5555** [comment]
> Shouldn't we also have Trident's positions for z_tilt in here as well?

Probably a wise idea, I haven't built a Trident yet, so I don't know much about it's configs. Happy to work with you to write it up if you willing to provide examples of your config so I can calculate the different size tridents.

**Comment by skyaee** [comment]
> > 我们不应该在这里也有三叉戟的 z_tilt 位置吗？
> 
> 可能是个明智的主意，我还没有构建 Trident，所以我不太了解它的配置。如果您愿意提供您的配置示例，我很高兴与您一起编写它，以便我可以计算不同大小的三叉戟。

[z_tilt]
##  Use Z_TILT_ADJUST to level the bed .
##  z_positions: Location of toolhead

##--------------------------------------------------------------------
## Uncomment below for 250mm build
#z_positions:
#    -50, 18
#    125, 298
#    300, 18
#points:
#    30, 5
#    125, 195
#    220, 5

## Uncomment below for 300mm build
#z_positions:
#   -50, 18
#   150, 348
#   350, 18
#points:
#   30, 5
#   150, 245
#   270, 5

## Uncomment below for 350mm build
#z_positions:
#   -50, 18
#   175, 398
#   400, 18
#points:
#   30, 5
#   175, 295
#   320, 5
